IoT_master/tools/desks_reservation_func.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import print_function
+import pickle
+import os.path
+import yaml
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+from os import path
+
+# If modifying these scopes, delete the file token.pickle.
+SCOPES = ['https://www.googleapis.com/auth/calendar']
+
+
+def calendar_config():
+    '''
+    Open a configuration file and read iframe,
+    file with credentials for Calendar API and
+    calendarID
+    :return: List with iframe, api_credentials, calendarID strings
+    :rtype: List
+    '''
+    if(path.exists("../config.yml")):
+        with open('../config.yml', 'r') as file_config:
+            content = yaml.load(file_config)
+        if 'calendar' in content:
+            iframe = content['calendar'].get('iframe', None)
+            api_credentials = content['calendar'].get('api_credentials', None)
+            calendarID = content['calendar'].get('calendarID', None)
+            desks = content['calendar'].get('desks', None)
```

Comment:
.get() already returns None by default if there's no key

Reply:
So you suggest that I should remove 'None' in every get() function call?

Reply:
Yes, because it’s already default